### PR TITLE
(fleet) remove all changes to the datadog-agent user/folders in the installer deb

### PR DESCRIPTION
**Pull request by arbll**
This PR removes all changes to datadog-agent user/folders that were previously made by the installer. Since we are currently not installing the agent those conflict with the agent deb scripts that do the same.

### CODE REVIEW

**Review by BaptisteFoy — APPROVED** [review]

File: omnibus/package-scripts/installer-deb/postinst [hunk 1]
```diff
@@ -36,53 +38,23 @@ set -e
...
-fi
-
-# Set helper rights
-chmod 750 ${HELPER}
+chmod 777 ${PACKAGES_LOCK_DIR}
 
-$BOOTSTRAP_INSTALLER bootstrap --package "datadog-installer"
-$STABLE_INSTALLER bootstrap
+# we currently allow failures to install the installer
+(bootstrap) || true
```
Comment (chouquette) [inline omnibus/package-scripts/installer-deb/postinst:58]: why is the command executed in a subshell?

File: omnibus/package-scripts/installer-deb/preinst [hunk 2]
```diff
@@ -3,7 +3,4 @@
 #
 # .deb: STEP 2 of 5
 
-
-SYSTEMCTL_SKIP_SYSV=true systemctl stop datadog-installer || true
```
Comment (chouquette) [inline omnibus/package-scripts/installer-deb/preinst:7]: This doesn't seem related to the agent itself nor its users, could you clarify why it's needed?

Comment (arbll) [reply omnibus/package-scripts/installer-deb/preinst:7]: We aren't currently installing the systemd unit for the installer and this will likely be done inside of the installer instead. 
We could leave it but given that we've hit a few issues with the QA already I want to trim down anything that could be an issue for step1

Comment (chouquette) [reply omnibus/package-scripts/installer-deb/preinst:7]: Fair enough, thanks for clarifying :)

**Review by chouquette — APPROVED** [review]

File: omnibus/package-scripts/installer-deb/postinst (hunk 1 above)
Comment (chouquette) [reply omnibus/package-scripts/installer-deb/postinst:58]: it should, but I wouldn't bother too much with it, this can be merged as is

File: omnibus/package-scripts/installer-deb/postinst [hunk 3]
```diff
@@ -36,53 +38,26 @@ set -e
 case "$1" in
     configure)
         add_user_and_group 'dd-installer' $PACKAGES_DIR
-        add_user_and_group 'dd-agent' $PACKAGES_DIR/datadog-agent
-        usermod -aG dd-agent dd-installer
```
Comment (raphaelgavache) [inline omnibus/package-scripts/installer-deb/postinst:40]: not being in the agent group will prevent writing logs and touch other dirs owned by agent

Comment (raphaelgavache) [reply omnibus/package-scripts/installer-deb/postinst:40]: Actually since the installer is run as root and changes it's groups manually, including dd-agent it should work

Comment (arbll) [reply omnibus/package-scripts/installer-deb/postinst:40]: It does from naive testing, I'll double check on a brand new VM
